Datadog.Trace.Samples.g.sln

```diff
@@ -447,6 +451,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Samples.Quartz", "tracer\te
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Samples.AzureFunctions.V4Isolated.HostLogsDisabled", "tracer\test\test-applications\azure-functions\Samples.AzureFunctions.V4Isolated.HostLogsDisabled\Samples.AzureFunctions.V4Isolated.HostLogsDisabled.csproj", "{C770F9F8-0430-587D-EB7A-8BEC2FE9B61C}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Samples.FeatureFlags", "tracer\test\test-applications\feature-flags\Samples.FeatureFlags\Samples.FeatureFlags.csproj", "{D5A8ABB9-CB23-974E-F338-5D9172D96CD8}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Samples.OpenFeature-2.9", "tracer\test\test-applications\feature-flags\Samples.OpenFeature-2.9\Samples.OpenFeature-2.9.csproj", "{021D714E-1764-D76B-15B5-C9114FE934C2}"
+EndProject
```

Comment:
The naming doesn't seem right here? Samples.FeatureFlags uses `OpenFeature` as well, right?, plus we should be doing multi-version testing with `OpenFeature`, so having a fixed version in the name doesn't really make sense

So tl;dr;
- Drop the `feature-flags` folder
- Consider merging the two samples
- Rename to `Samples.OpenFeature`

Reply:
Samples.FeatureFlags does not use OpenFeature. It uses the exposed SDK functions directly